.gitignore

```diff
@@ -31,9 +31,13 @@ tests/data/common_wheels/
 # Misc
 *~
 .*.sw?
+.env/
 
 # For IntelliJ IDEs (basically PyCharm)
 .idea/
 
+# For Visual Studio Code
+.vscode/
```

Comment:
I'm tempted to suggest that you add this in your own ~/.gitconfig but then, instead of making all contributors do it, I guess this is better?

Reply:
Given the existence of `.idea/` in the line above, I don't see that it's a problem.